spec/requests/api/v1/passwords/update_spec.rb

```diff
@@ -0,0 +1,123 @@
+require 'rails_helper'
+
+RSpec.describe 'PATCH /api/v1/users/password', type: :request do
+  let(:user) { create(:user) }
+  let(:password_token) { user.send(:set_reset_password_token) }
+  let!(:headers) do
+    params = {
+      reset_password_token: password_token,
+      redirect_url: ENV['PASSWORD_RESET_URL']
+    }
+    get edit_user_password_path, params: params, headers: auth_headers
+    edit_response_params = Addressable::URI.parse(response.header['Location']).query_values
+    {
+      'access-token' => edit_response_params['token'],
+      'uid' => edit_response_params['uid'],
+      'client' => edit_response_params['client_id']
+    }
+  end
+
+  subject { patch user_password_url, params: params, headers: headers, as: :json }
+
+  context 'when the request is valid' do
+    let(:params) do
+      {
+        "password": 'Newpass123',
```

Comment:
I think you can just put `password: 'Newpass123'` (you can avoid putting the double quotes for key values)